client/src/components/Address/Address.js

```diff
@@ -164,12 +175,8 @@ const Address = () => {
           .then((res) => {
             postOrderItems(ordersID)
               .then(async (res) => {
-                if (res.data.message == "Sucessfully added order items!!") {
-                  await deleteCart(cartItems.cart_id);
-                  navigate("/order/confirm");
-                } else {
-                  navigate("/order/error");
-                }
+                await deleteCart(cartItems.cart_id);
```

Comment:
what will happen if this async call fails?
are you handling the error case well?

Reply:
this I will remove and start from the scratch,as we change db design now.